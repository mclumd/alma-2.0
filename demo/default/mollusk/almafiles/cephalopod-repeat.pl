rel(cephalopod, steve).
if(now(2), rel(cephalopod, sonny)).
if(now(4), rel(cephalopod, sam)).
if(now(6), rel(cephalopod, sue)).
