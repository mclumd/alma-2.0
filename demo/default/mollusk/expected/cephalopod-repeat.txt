~rel(shell_bearer, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
rel(mollusk, steve)
rel(cephalopod, steve)
~rel(shell_bearer, sonny)
abnormal(sonny, mollusk, "~rel(shell_bearer, sonny)")
rel(mollusk, sonny)
rel(cephalopod, sonny)
~rel(shell_bearer, sam)
abnormal(sam, mollusk, "~rel(shell_bearer, sam)")
rel(mollusk, sam)
rel(cephalopod, sam)
~rel(shell_bearer, sue)
abnormal(sue, mollusk, "~rel(shell_bearer, sue)")
rel(mollusk, sue)
rel(cephalopod, sue)
